<launch>
    <arg name="robot_base_frame"        default="base_footprint"/>
    <arg name="odom_frame"              default="odom"/>
    <arg name="map_frame"               default="map"/>
    <arg name="scan_frame"              default="base_scan"/>

    <node pkg="costmap_2d" type="costmap_2d_node" name="local_costmap">
        <rosparam file="$(find rt_obstacle_layer)/param/single_local_costmap_params.yaml" command="load"/>
        <param name="~costmap/global_frame"  value="$(arg robot_base_frame)"/>
        <param name="~costmap/robot_base_frame"  value="$(arg robot_base_frame)"/>
    </node>
</launch>
